### ISSUE TITLE
ways to get the heating duration and the user defined temperatures

### ISSUE THREAD

**Issue by dedelajoie** [KiboOst/php-qivivoAPI#3]
Hello, this is not a bug report but actually 2 questions : 

1) on the COMAP web site in the user interface there is a [Statistiques] tabs with very interesting data. Among them the duration time of eating (per day, weeks,...). Do you know if there is a way to get this value ? 

2) is there a way to read the temperature settings ? (I can get the temperature of the sensor module, the temperature I am setting but not the ones that I programmed)

And again and above all thank you and congratulations for your amazing work !

**Comment by KiboOst** [comment]
Hi,
1: For sure there is a way to get them, but it's not implemented into the API. The main goal of the API is to provide actions/infos for Jeedom plugin, and Jeedom histories all data and display them graphically, so there is no need for it into the API.

2: Yes, this was previously reported by the API, but things have changed on comap side. I've just fix it yesterday on Jeedom plugin. I've also just updated the API on this repo.
Basically, you need to get `$_qivivo->getFullDevices()` and check for model thermostat, order should be correct now.

**Comment by dedelajoie** [comment]
Thank you again for your great work ! 
So in your opinion without using Jeedom what would be the best way to assess the duration of eating ? 
I was thinking about testing the state of the heating_status on a regular basis (like every 10 minutes or so) but I am sure there is a more elegant way to do so. No ?

**Comment by KiboOst** [comment]
Yes you can do a cron that check state, but best way for me would be to integrate this in the API. Not much time right now, but shouldn't be hard, you just need to ask the right url and parse the result, adding a getStats() function.

**Comment by dedelajoie** [comment]
Thank you again for you answers ! But I don't have much knowledge on how to do that. 
I know how to handle API response but I don't get the how the identification process. 
More generally is there any place where I could find any kinf of documentation regarding this API ?
I really appreciate the time you dedicate to us and the quality of your work !
